cognite/client/data_classes/workflows.py

```diff
@@ -1175,3 +1175,146 @@ def load(cls, resource: Any, cognite_client: CogniteClient | None = None) -> Wor
 
     def dump(self, camel_case: bool = True, as_external_id: bool = False) -> list[dict[str, Any]]:
         return [workflow_id.dump(camel_case, as_external_id_key=as_external_id) for workflow_id in self.data]
+
+
+WorkflowTriggerType: TypeAlias = Literal["scheduledTrigger"]
+
+
+class WorkflowScheduledTriggerRule(CogniteObject):
+    """
+    This class represents a scheduled trigger rule.
+
+    Args:
+        cron_spec (str | None): The cron specification for the scheduled trigger.
+    """
+
+    def __init__(self, cron_spec: str | None = None) -> None:
+        self.type: WorkflowTriggerType = "scheduledTrigger"
+        # self.trigger_type: WorkflowTriggerType = "scheduledTrigger"
+        self.cron_spec = cron_spec
+
+
+class WorkflowTrigger(CogniteResource):
```

Comment:
I'm not a big fan of the shared Read and Write class, may be confusing for users of the create endpoint.